### PR TITLE
1604 by Inlead: Implement ding_sections to DDB CMS.

### PR DESCRIPTION
**Pull request by bozeak**
http://platform.dandigbib.org/issues/1604

### CODE REVIEW

**Comment by JrgenGNielsen** [comment]
reviewed and approved

File: modules/ding_sections/modules/ding_sections_term_menu/ding_sections_term_menu.install [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  if (!db_table_exists($table)) {
+    db_create_table($table, drupal_get_schema_unprocessed('ding_sections_term_menu', $table));
+    return 'Add the ding_sections_term_menu table for the ding_sections_term_menu module.';
+  }
+}
+
+/**
+ * Create relations between editorial base term ids to section menu items.
+ */
+function ding_sections_term_menu_update_7002() {
```
[the remark this answers is not in the document]
Comment (bozeak) [reply modules/ding_sections/modules/ding_sections_term_menu/ding_sections_term_menu.install]: At the moment this in not needed here. Removed.

File: modules/ding_sections/modules/ding_sections_term_panel/ding_sections_term_panel.module [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+    // When unchecked.
+    if (($term_page[0]['value'] - $term_page_old[0]['value']) == -1) {
+      // Get page.
+      $page = 'term_view';
+      $page = page_manager_get_page_cache($page);
+      // @see page_manager.admin.inc
+      if (!empty($page->handler_info['term_view_panel_context_tid_' . $term->tid])) {
+        $page->handler_info['term_view_panel_context_tid_' . $term->tid]['disabled'] = TRUE;
+        $page->handler_info['term_view_panel_context_tid_' . $term->tid]['changed'] |= PAGE_MANAGER_CHANGED_STATUS;
```
[the remark this answers is not in the document]
Comment (bozeak) [reply modules/ding_sections/modules/ding_sections_term_panel/ding_sections_term_panel.module]: https://cgit.drupalcode.org/ctools/tree/page_manager/page_manager.module#n29
page_manager defines this constant as a bit flag, so it should be operated correctly by using such construction.

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**